### ISSUE TITLE
Certain file types not accepted for uploads

### ISSUE THREAD

**Issue by svenvarkel** [balderdashy/skipper#90]
I'm trying to get file uploads working with SailsJS/Skipper. The files I want to use are BDOC containers (that's a standard for containers for digital signatures).
When I try to upload a BDOC file to my SailsJS app I get the error:

        _multipartyError: [Error: unrecognized content-type: application/bdoc]

How to make this content type "recognized" by SailsJS/Skipper?

**Comment by sailsbot** [comment]
Thanks for posting, @svenvarkel.  I'm a repo bot-- nice to meet you!

It has been 60 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
